ajax/libs/blendui/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "blendui",
+  "filename": "BlendHybridUI.min.js",
+  "author": "Clouda+ team",
+  "description": "A Hybrid Javascript Framework For Mobile WebApp",
+  "homepage": "http://clouda.baidu.com/blendui/introduction/introduction",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Clouda-team/BlendUI.git"
+  },
+  "version": "0.0.4",
+  "keywords": [
+    "Native",
+    "Javascript",
+    "webview",
+    "web"
+  ],
+  "npmName": "blendui",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "!(BlendHybridUI-*)"
```

Comment:
Haven't you read the doc? Why not just use RegExp?

https://github.com/isaacs/node-glob#glob-primer 

> `[...]` Matches a range of characters, similar to a RegExp range. If the first character of the range is `!` or `^` then it matches any character not in the range.